docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-promo-block-internal.html.twig

```diff
@@ -0,0 +1,78 @@
+{% if content.field_featured_item[0] %}
```

Comment:
You should not need so much usage of `[0]` here. 

Also all the comments about the dropdowns and not rendering the fields in order to draw them apply here.